cypress/support/index.js

```diff
@@ -20,10 +20,13 @@ import './commands';
 // require('./commands')
 
 Cypress.on('uncaught:exception', err => {
+  console.log('Cypress detected uncaught exception', err.name);
   // Rapidly cy.visiting pages seems to cause an uncaught exception. This seems
   // to be a testing artifact, since users can't click fast enough to cause this
   // (to our knowledge).
-  if (err.name === 'ChunkLoadError') {
+  // UPDATE: this may have morphed into NS_ERROR_UNEXPECTED with the move to
+  // Cypress 9.  Quite why, I'm not sure.
+  if (err.name === 'NS_ERROR_UNEXPECTED') {
```

Comment:
```suggestion
  if (err.name === 'NS_ERROR_UNEXPECTED' || err.name === 'ChunkLoadError') {
```

Should we include both just to be safe?

Reply:
I'd rather not, honestly.  We can if Cypress starts failing because of them, but the fewer things we ignore the more useful the tests become.